src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int numberOfBalls = 3;
+        Lottery lottery = new Lottery();
+        Ball [] balls = new Ball[numberOfBalls];
+        for (int i = 0; i < numberOfBalls; i++) {
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i].toString());
```

Comment:
we don't really need this array here, you are filling it and getting in the same place

Reply:
That seemed to me more interesting than just sout three balls by three rows, but ok.